src/applications/vaos/components/AppointmentRequestListItem.jsx

```diff
@@ -117,6 +118,16 @@ export default class AppointmentRequestListItem extends React.Component {
               </div>
             </div>
 
+            {firstMessage && (
+              <div className="vaos_appts__message vads-u-flex--1 vads-u-margin-y--2">
+                <dl className="vads-u-margin--0">
+                  <dt className="vads-u-font-weight--bold">
+                    Additional Information
```

Comment:
"Information" should be lower case.

This also doesn't seem to match with the mocked screenshot in the issue, but I'm not sure its feasible to fetch all the messages up front, rather than behind the "Show more" link.